staging/src/k8s.io/apimachinery/pkg/apis/meta/v1beta1/types.go

```diff
@@ -133,6 +133,10 @@ const (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 type TableOptions struct {
 	v1.TypeMeta `json:",inline"`
+
+	// NoHeaders is only exposed for internal callers.
+	NoHeaders bool `json:"-"`
```

Comment:
o_O

Reply:
Other option includes passing internal around everywhere but we can't really do that on some of the calls easily.  I couldn't think of anything better...